packages/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerButtonViewManager.kt

```diff
@@ -200,7 +200,7 @@ class RNGestureHandlerButtonViewManager : ViewGroupManager<ButtonViewGroup>(), R
 
     var exclusive = true
 
-    private var _backgroundColor = Color.TRANSPARENT
+    private var backgroundColor = Color.TRANSPARENT
```

Comment:
IIRC this was on purpose so as not to conflict with View's `backgroundColor`. It doesn't necessarily mean than it needs the underscore but I don't think we can keep it as `backgroundColor`.

Reply:
So I guess the same goes with `_enabled`? We can't keep `_` as the prefix as the linter throws error, but we may change to to something like `buttonBackgroundColor`, what about that?

Reply:
Sounds good, add a comment about avoiding name conflict there.

Reply:
Okay, I've changed `backgroundColor` in [dff404d](https://github.com/software-mansion/react-native-gesture-handler/pull/3507/commits/dff404d902c6c90a74c5dd2aea994d86b980a656) and `enabled` in [585ac44](https://github.com/software-mansion/react-native-gesture-handler/pull/3507/commits/585ac44e2472fbe4cf6b7920dd445327147602fe)